spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/EntityGraph.java

```diff
@@ -71,7 +70,7 @@
 	 * @author Thomas Darimont
 	 * @since 1.6
 	 */
-	public enum EntityGraphType {
+	enum EntityGraphType {
 
```

Comment:
In general, don't alter visibility settings without first consulting with the team. We have a strict policy to avoid breaking changes and generally discuss changes in visibility before carrying them out.

Reply:
Actually, this modifier has been redundant here because the enum is declared inside an annotation definition. So, this is not a breaking change.